src/Rules/PhpDoc/RequireExtendsDefinitionClassRule.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\PhpDoc;
+
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\Node\InClassNode;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleErrorBuilder;
+use function count;
+
+/**
+ * @implements Rule<InClassNode>
+ */
+class RequireExtendsDefinitionClassRule implements Rule
+{
+
+	public function __construct(
+		private RequireExtendsCheck $requireExtendsCheck,
+	)
+	{
+	}
+
+	public function getNodeType(): string
+	{
+		return InClassNode::class;
+	}
+
+	public function processNode(Node $node, Scope $scope): array
+	{
+		$classReflection = $node->getClassReflection();
+		$extendsTags = $classReflection->getRequireExtendsTags();
+
+		if (count($extendsTags) === 0) {
```

Comment:
I think we should error when a class defines 2 or more `@phpstan-require-extends`

Reply:
Not error, but support and check all of them.

Reply:
Oh sorry, I misunderstood you. Please add a test and fix the rule.